smime-messaging/info.json

```diff
@@ -0,0 +1,135 @@
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "file",
+        "name": "public_key",
+        "tooltip": "Specify the Public key to access the S/MIME Messaging to which you will connect and perform the automated operations"
+      },
+      {
+        "title": "Private Key",
+        "description": "Specify the Private key to access the S/MIME Messaging to which you will connect and perform the automated operations",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "file",
+        "name": "private_key",
+        "tooltip": "Specify the Private key to access the S/MIME Messaging to which you will connect and perform the automated operations"
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "sign_email",
+      "title": "Sign Email",
+      "description": "Generate an S/MIME-signed message.",
+      "category": "investigation",
+      "annotation": "sign_email",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Message Body",
+          "description": "Specify the message body to send.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "body"
+        }
+      ],
+      "output_schema": {
+        "SMIME.Signed.Email": {
+          "message": ""
+        }
+      }
+    },
+    {
+      "operation": "verify_sign",
+      "title": "Verify Sign",
+      "description": "Verifies the signature of the message. Note: Specify the signed email with .p7 extension",
+      "category": "investigation",
+      "annotation": "verify_sign",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "File IRI/Path",
+          "description": "Specify the file IRI/path where the file is located. File IRI is an attribute of a file which is in turn an attribute of an attachment or an indicator. NOTE: The file name is taken from the tmp directory.",
+          "name": "file_iri",
+          "type": "text",
+          "visible": true,
+          "editable": true,
+          "required": true,
+          "tooltip": "Specify the file IRI/path where the file is located. File IRI is an attribute of a file which is in turn an attribute of an attachment or an indicator. NOTE: The file name is taken from the tmp directory."
+        }
+      ],
+      "output_schema": {
+        "result": ""
+      }
+    },
+    {
+      "operation": "encrypt_email",
+      "title": "Encrypt Email",
+      "description": "Generate an S/MIME-encrypted message",
+      "category": "investigation",
+      "annotation": "encrypt_email",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Message Body",
+          "description": "Specify the message body to encrypt.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
```

Comment:
Please update type for Message Body parameter to textarea or richtext(if possible) in Encrypt Email and Sign Email actions.